plugins/module_utils/_SqlServerUtils.psm1

```diff
@@ -139,6 +139,13 @@ function ConvertTo-SerializableObject {
             $pValue = $p.Value
 
             switch ($p) {
+                { $null -eq $pValue } {
```

Comment:
Surprisingly it wasn't until now I encountered a $null value being returned by DbaTools (LastLoginTime) - this should handle that use case.